actiontext/app/helpers/action_text/tag_helper.rb

```diff
@@ -11,7 +11,8 @@ module TagHelper
     # that Trix will write to on changes, so the content will be sent on form submissions.
     #
     # ==== Options
-    # * <tt>:class</tt> - Defaults to "trix-content" which ensures default styling is applied.
+    # * <tt>:class</tt> - Defaults to "trix-content" so that default styles will be applied. Setting 
+        this to a different value will prevent default styles from being applied.
```

Comment:
@stevepolitodesign Missing a comment mark here, but otherwise these changed look good to me.

```suggestion
    #   this to a different value will prevent default styles from being applied.
```

After you squash, I'll give this a merge. :+1: